pages/code-of-conduct.tsx

```diff
@@ -10,89 +10,175 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
   >
     <h1>Code of Conduct</h1>
 
+    <h2>Purpose</h2>
     <p>
-      All attendees, speakers, sponsors and volunteers at our conference are required to agree with the following code
-      of conduct. We are expecting cooperation from all participants to help ensuring a safe, welcoming and inclusive
-      environment for everybody.
+      {props.pageMetadata.conference.Name} is dedicated to providing a harassment-free conference experience for
+      everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical
+      appearance, body size, race, age or religion. We do not tolerate harassment of conference participants in any
+      form. Sexual language and imagery is not appropriate for any conference venue, including talks.
     </p>
 
-    <p
-      style={{
-        fontSize: '150%',
-        textAlign: 'center',
-        textDecoration: 'underline',
-      }}
-    >
-      <strong>Organisers will enforce this code throughout the event.</strong>
+    <p>
+      Conference participants or anyone interacting with the event violating these rules may be sanctioned or expelled
+      from the conference without a refund at the discretion of the conference organisers.
     </p>
 
-    <h2>Need Help?</h2>
+    <h2>Anti-harassment</h2>
+
+    <p>Harassment includes, but is not limited to:</p>
+
+    <ul>
+      <li>
+        Verbal or written comments that reinforce social structures of domination related to gender, gender identity and
+        expression, sexual orientation, disability, physical appearance, body size, race, age, religion
+      </li>
+      <li>Sexual images in public spaces</li>
+      <li>Deliberate intimidation, stalking, or following</li>
+      <li>Harassing photography or recording</li>
+      <li>Sustained disruption of talks or other events</li>
+      <li>Inappropriate physical contact</li>
+      <li>Unwelcome sexual attention</li>
+      <li>Advocating for, or encouraging, any of the above behaviour</li>
+    </ul>
+
+    <h2>Enforcement</h2>
+
+    <p>Participants asked to stop any harassing behavior are expected to comply immediately.</p>
+
     <p>
-      Email <a href={props.pageMetadata.conference.ContactEmail}>{props.pageMetadata.conference.ContactEmail}</a> or
-      call our designated contact person {props.pageMetadata.conference.EmergencyContactName} on{' '}
-      {props.pageMetadata.conference.EmergencyContactPhoneNumber}. You can also approach any one of the conference staff
-      who will be identifiable by the branded DDD Perth t-shirts they'll be wearing.
+      If a participant engages in harassing behaviour, event organisers retain the right to take any actions to keep the
+      event a welcoming environment for all participants. This includes warning the offender or expulsion from the
+      conference with no refund.
     </p>
```

Comment:
DMs are open on twitter. You may wish to note who sees them, being Rob and I. It's useful I think to provide an option that is seen by a woman.
Also noting that any contact regarding the CoC will be treated discreetly.